templates/controller-mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if .Values.controller.enabled }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ template "consul.fullname" . }}-controller-mutating-webhook-configuration
+  labels:
+    app: {{ template "consul.name" . }}
+    chart: {{ template "consul.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: controller
+  annotations:
+    cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ template "consul.fullname" . }}-controller
+webhooks:
+- clientConfig:
+    caBundle: Cg==
+    service:
+      name: {{ template "consul.fullname" . }}-controller-webhook
+      namespace: {{ .Release.Namespace }}
+      path: /mutate-consul-hashicorp-com-v1alpha1-servicedefaults
+  failurePolicy: Fail
+  name: mservicedefaults.kb.io
```

Comment:
```suggestion
  name: mservicedefaults.consul.io
```

Reply:
We could make this hashicorp.com as well

Reply:
I made it `mutate-servicedefaults.consul.hashicorp.com`

Reply:
Will update the other PR with the same